lib/linux_admin/chrony.rb

```diff
@@ -0,0 +1,21 @@
+module LinuxAdmin
+  class Chrony
+    def initialize(conf = "/etc/chrony.conf")
+      raise MissingConfigurationFileError, "#{conf} does not exist" unless File.exist?(conf)
+      @conf = conf
+    end
+
+    def clear_servers
+      data = File.read(@conf)
+      data.gsub!(/server\s+.+\n/, "")
+      File.write(@conf, data)
+    end
+
+    def add_servers(*servers)
+      data = File.read(@conf)
+      data << "\n" unless data.end_with?("\n")
```

Comment:
I couldn't decide what the best way to deal with this was.
I wanted to deal with the case where the file didn't end with a newline, but also make sure the file did end with a newline when I was finished with it.